src/Chapter2.hs

```diff
@@ -336,7 +338,13 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList _ _ [] = []
+subList m n (x:xs) 
+  | m < 0 || n < 0 = []
+  | m > n = []
+  | m == n = [x]
```

Comment:
I'm not sure this case is valid :thinking: If `m == n == 3` then you need to take the fourth element of the list, not the current one in the head of a list